src/vellum/client/core/pydantic_utilities.py

```diff
@@ -304,3 +304,79 @@ def _get_field_default(field: PydanticField) -> typing.Any:
             return None
         return value
     return value
+
+
+def convert_pydantic_to_json_schema(schema_input: typing.Any) -> typing.Dict[str, typing.Any]:
```

Comment:
we should never add logic to the `src/vellum/client` directory. This directory is auto-generated.

Instead, move whatever we need somewhere `src/vellum/workflows`

Reply:
Delete the changes in src/vellum/client files